test/FsAutoComplete.Tests.Lsp/Tests.fs

```diff
@@ -734,6 +735,142 @@ let foldingTests =
     ))
   ]
 
+
+let inline waitForParsedScript (m: System.Threading.ManualResetEvent) (event: Event<string * obj>) =
+
+  let bag = new System.Collections.Concurrent.ConcurrentBag<LanguageServerProtocol.Types.PublishDiagnosticsParams>()
+
+  event.Publish
```

Comment:
Hah! I've made similar helper functions over in #488, great minds think alike :)